src/app/modules/item/components/chapter-children/chapter-children.component.html

```diff
@@ -2,9 +2,41 @@
 
 <p *ngIf="state === 'error'">Error while loading the group managers</p>
 
-<ng-container *ngIf="state === 'ready'">
-  <p class="validation-text">
+<ng-container *ngIf="state !== 'loading' && state !== 'error'">
+  <br>
+  <p class="validation-text" *ngIf="state === 'ready-missing-validation'">
     To validate this chapter, solve at least all tasks with Validation type
   </p>
-  Ready, loaded {{children.length}} children.
+  <p class="validation-text" *ngIf="state === 'empty'">
+    This chapter has no content visible to you, so you can't validate it for now.
+  </p>
+
+  <ul class="data-list-container activity">
```

Comment:
Probably not useful to display it is empty (in case the "ul" takes some place)

Reply:
fixed